sim/battle.ts

```diff
@@ -2987,6 +2989,24 @@ export class Battle extends Dex.ModdedDex {
 		return false;
 	}
 
+	hint(hint: string, once?: boolean, sideid?: 'p1'|'p2') {
```

Comment:
```suggestion
	hint(hint: string, once?: boolean, sideid?: 'p1' | 'p2') {
```

Here's your Travis error.